packages/runtime/src/types/transport/RelationshipDTO.ts

```diff
@@ -11,11 +10,28 @@ export enum RelationshipStatus {
     Terminated = "Terminated"
 }
 
+export enum AuditLogEntryReason {
+    Creation = "Creation",
+    AcceptanceOfCreation = "AcceptanceOfCreation",
+    RejectionOfCreation = "RejectionOfCreation",
+    RevocationOfCreation = "RevocationOfCreation"
+}
+export interface AuditLogEntryDTO {
+    createdAt: string;
+    createdBy: string;
+    reason: AuditLogEntryReason;
+    oldStatus?: RelationshipStatus;
+    newStatus: RelationshipStatus;
+}
+
+export interface AuditLogDTO extends Array<AuditLogEntryDTO> {}
+
 export interface RelationshipDTO {
     id: string;
     template: RelationshipTemplateDTO;
     status: RelationshipStatus;
     peer: string;
     peerIdentity: IdentityDTO;
-    changes: RelationshipChangeDTO[];
```

Comment:
since the changes are not written in the runtime, the type extension can be pushed to where this actually happens - I think it's cleaner

Reply:
wait what? 😆 The main point of DTOs is to prevent backwards incompatible changes.

you have to "mock" this changes array inside the dto and create it in the mapper. (only containing the creation change)

Reply:
What about the DVO?

Reply:
DVOs are only used by the app and we can adapt to changes way easier there than the connector users.

Reply:
and regarding backwards compatibility, the response source switching from relationship change to relationship is not an issue?